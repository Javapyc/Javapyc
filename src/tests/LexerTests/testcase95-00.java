System.out.println
this.foo