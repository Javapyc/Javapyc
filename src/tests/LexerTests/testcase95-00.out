ReservedWord, System.out.println
ReservedWord, this
Delimiter, .
ID, foo
